src/main/java/com/codeborne/selenide/webdriver/ChromeDriverFactory.java

```diff
@@ -57,7 +57,7 @@ public MutableCapabilities createCapabilities(Config config, Browser browser,
 
     ChromeOptions options = new ChromeOptions();
     options.setHeadless(config.headless());
-    if (!config.browserBinary().isEmpty()) {
+    if (config.browserBinary() != null) {
```

Comment:
I don't see why this change is useful.
if config.browserBinary() is empty string, the following line will not work anyway.

Reply:
@BorisOsipov ^^^